src/App.tsx

```diff
@@ -1,34 +1,48 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
+import { useDispatch } from 'react-redux';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+import { actions } from './features/todos';
+import { useAppSelector } from './app/hooks';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [isTodoLoading, setIsTodoLoading] = useState(true);
+  const dispatch = useDispatch();
+  const curentTodo = useAppSelector(state => state.currentTodo);
+
+  useEffect(() => {
+    getTodos()
+      .then(data => dispatch(actions.setTodos(data)))
+      .finally(() => setIsTodoLoading(false));
```

Comment:
In asynchronous requests, it is always necessary to assume that the request will end with an error